template_project/app/build.gradle

```diff
@@ -148,7 +148,7 @@ task generateShorcutsFile {
                     'intent'(
                             'android:action': 'android.intent.action.MAIN',
                             'android:targetPackage': twaManifest.applicationId,
-                            'android:targetClass': 'android.support.customtabs.trusted.LauncherActivity',
+                            'android:targetClass': 'com.google.androidbrowserhelper.trusted.LauncherActivity',
```

Comment:
You've tested this, right?

Reply:
Yes, tested against `https://contentindex.dev/manifest.json`. It works :)

Reply:
Does the previous value not work though?  It's still working for me

Reply:
Are you using the `android-browser-helper-1.1.0`? The old class doesn't exist anymore in `androidx.browser`, so I wouldn't expect it to work.